docs/docs/guides/net_migration_guide.md

```diff
@@ -0,0 +1,15 @@
+# Web3 Net Migration guide
+
+## Breaking Changes
+
+### getId
```

Comment:
Its already mentioned in breaking changes section but, I think we should also add an example in main Web3 package migration guide doc:

`let web3 = new Web3(Web3.givenProvider);`

It will not have:
```
// -> web3.bzz.net
// -> web3.shh.net
```